azure/cosmo_db/terraform.tf

```diff
@@ -0,0 +1,15 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~> 3.6"
+    }
+
+    azurecaf = {
+      source  = "aztfmod/azurecaf"
+      version = "2.0.0-preview3"
+    }
+  }
+
+  required_version = ">= 1.3.0, < 2.0.0"
+}
```

Comment:
Should contain less see kv example - we dropped caf

Reply:
The version suggested is to support the new features of terraform, like optional parameters. It would be a good idea to keep it as version 1.3.0 like are doing in the new pull requests

Reply:
i will remove the caf part atleast for now and keep the versions

Reply:
This one is now resolved @merou